lib/DDG/Spice/Timer.pm

```diff
@@ -13,12 +22,19 @@ handle remainder => sub {
     my $qry = $_;
     my $raw = lc($req->query_raw);
```

Comment:
It would probably simplify the regex if we use `query_lc` or `query_clean`

Then we don't need to account for letter casing or whitespace variations